packages/plugin-aws-lambda/test/index.test.ts

```diff
@@ -738,4 +738,36 @@ describe('plugin: aws lambda', () => {
     expect(events).toHaveLength(0)
     expect(sessions).toHaveLength(1)
   })
+
+  it('notifies when an string is passed (callback)', async () => {
```

Comment:
```suggestion
  it('supports a string as the error argument in a lambda callback, async () => {
```

I was fixing the grammatical `a/an` issue with this test description and realised it could do with a more descriptive name entirely.